valhalla/jawn/src/lib/clients/kafkaConsumers/KafkaConsumer.ts

```diff
@@ -148,6 +150,27 @@ function mapKafkaMessageToMessage(
   return ok(messages);
 }
 
+function mapKafkaMessageToScoresMessage(
+  kafkaMessage: KafkaMessage[]
+): GenericResult<HeliconeScoresMessage[]> {
+  const messages: HeliconeScoresMessage[] = [];
+  for (const message of kafkaMessage) {
+    if (message.value) {
+      try {
+        const kafkaValue = JSON.parse(message.value.toString());
+        const parsedMsg = JSON.parse(kafkaValue.value) as HeliconeScoresMessage;
```

Comment:
Careful, when dates are serialized to JSON and then deserialized they are not a `Date` object, but rather a string. If you are expecting a date object you might need to do something like `new Date` for the createdAt field... if you are even using it that is

Reply:
I'm doing this right now before sending to Kafka 

```
const feedbackMessage: HeliconeScoresMessage = {
      requestId: requestId,
      organizationId: this.authParams.organizationId,
      scores: [
        {
          score_attribute_key: "helicone-score-feedback",
          score_attribute_type: "number",
          score_attribute_value: feedback ? 1 : 0,
        },
      ],
      createdAt: new Date(),
    };
```